packages/fwfh_chewie/pubspec.yaml

```
@@ -1,17 +1,17 @@
 name: fwfh_chewie
-version: 0.7.1+2
+version: 0.13.0-alpha.1
 description: WidgetFactory extension to render VIDEO with the chewie plugin.
 homepage: https://github.com/daohoangson/flutter_widget_from_html
 
 environment:
-  flutter: ">=2.5.0"
-  sdk: ">=2.14.0 <4.0.0"
+  flutter: ">=3.7.0"
+  sdk: ">=2.19.0 <4.0.0"
 
 dependencies:
   chewie: ^1.0.0
```

I have tested on my machine with local copy of your package. It works without any other changes

Here is [the issue created by me](https://github.com/daohoangson/flutter_widget_from_html/issues/945)
```suggestion
  chewie: ^1.5.0
```